apps/condo/domains/ticket/schema/CallRecordFragment.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Generated by `createschema ticket.CallRecordFragment 'ticket:Relationship:Ticket:CASCADE;callRecord:Relationship:CallRecord:CASCADE;'`
+ */
+
+const { Relationship, DateTimeUtc } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
+const access = require('@condo/domains/ticket/access/CallRecordFragment')
+
+
+const CallRecordFragment = new GQLListSchema('CallRecordFragment', {
+    schemaDoc: 'M2M relationship between ticket and call record',
+    fields: {
+        ticket: {
+            type: Relationship,
+            ref: 'Ticket',
+            knexOptions: { isNotNullable: false },
+        },
+        callRecord: {
+            type: Relationship,
+            ref: 'CallRecord',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+        startedAt: {
```

Comment:
Why need need this field, if it's already in `CallRecord` itself?

Reply:
This is the time at which fragment begins.
Call records log displays fragments and sorts by this field.

When you attach a `ticket` to `callRecord`, a new `callRecordFragment` is created with the same `startedAt` as `callRecord`

Reply:
Why not it's just offset from start then?

Reply:
We need to sort fragments by `startedAt` field (it's not equal `createdAt`)